docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
- [ ] Why not [use the source tree available here](https://github.com/docker-library/docs/tree/master/golang#compile-your-app-inside-the-docker-container)? [The `onbuild` image](https://github.com/docker-library/docs/tree/master/golang#golangonbuild) makes it even easier.